js/miss.js

```diff
@@ -0,0 +1,32 @@
+
+
+
+function onMissLaser() {
+    makeNoise('miss')
+    changeScore(-5)
+
+}
+
+function changeScore(value) {
+    var score = document.querySelector('#score')
+    score.textContent = parseInt(score.textContent) + value
+}
+
+function onHitCity(cityid) {
+    makeNoise('hitcity')
+    var explosion = document.querySelector('#city-' + cityid + ' .explosion')
+    if(explosion.getAttribute('src') === null) {
+        explosion.removeAttribute('src')
+        explosion.setAttribute('src', 'img/city-hit.gif')
+        var timeout = setTimeout( function (e) {
```

Comment:
whitespace

Reply:
Removed all unnecessary whitespace in ad42ddf